features/event-timing.yml

```
@@ -1,7 +1,9 @@
-draft_date: 2024-09-09
-name: Event Timing API
+name: Event timing
 description: TODO
```

For consistency with other features like this:

```suggestion
description: The `event` and `first-input` performance entries and the `PerformanceEventTiming` API measures the latency of user input events, such as mouse clicks or keypresses, and the latency of the first user input. First Input Delay (FID) is a common metric for perceived responsiveness.
```